src/meta-srv/src/region/supervisor.rs

```diff
@@ -526,4 +595,31 @@ pub(crate) mod tests {
             }
         }
     }
+
+    #[tokio::test]
+    async fn test_region_failure_detector_controller() {
+        let (mut supervisor, sender) = new_test_supervisor();
+        let controller = RegionFailureDetectorControl::new(sender.clone());
+        tokio::spawn(async move { supervisor.run().await });
+        let ident = (0, 1, RegionId::new(1, 1));
```

Comment:
still, some names like `ident`, maybe change them to `region`?

Reply:
Renamed in [40b24d5](https://github.com/GreptimeTeam/greptimedb/pull/4223/commits/40b24d501283203c23704a833d1a21c142533a75)